src/net_processing.cpp

```diff
@@ -2681,6 +2681,9 @@ class CompareInvMempoolOrder
     }
 };
 
+void MsgGetData(const Consensus::Params& consensusParams, CNode* pto, CConnman& connman, CNetMsgMaker& msgMaker, CNodeState& state, int64_t nNow, bool fFetch);
+void MsgFeeFilter(CNode* pto, CConnman& connman, CNetMsgMaker& msgMaker);
```

Comment:
Right, I didn't realise both versions of CNetMsgMaker::Make can be just made const too.
Updating both this and #9608.